src/shared/dropdown-list/types.ts

```diff
@@ -11,6 +12,8 @@ export interface ListItemDisplayProps {
 }
 
 export interface DropdownDisplayProps<T, V> {
+    /** Specifying flex direction within item */
+    secondaryLabelDisplayType?: LabelDisplayType | undefined;
```

Comment:
1. can move this down to the main `DropdownListProps` to avoid unintended inheritance in other select inputs
2. can the prop be renamed to `labelDisplayType` for consistency and also just to keep it short?

```suggestion
    labelDisplayType?: LabelDisplayType | undefined;
```